--- rtm_tcr_unittest.user.js	2012-03-21 14:52:29.000000000 -0400
+++ rtm_tcr.user.js	2012-03-21 14:50:21.000000000 -0400
@@ -1,10 +1,10 @@
 // ==UserScript==
-// @name           RTM: Tag Cloud Restructurer (rtm.tcr.unittest)
-// @namespace      http://www.rememberthemilk.com/home/rtm.tcr.unittest/*
-// @include        http://www.rememberthemilk.com/home/rtm.tcr.unittest/*
-// @include        https://www.rememberthemilk.com/home/rtm.tcr.unittest/*
-// @include        http://*.www.rememberthemilk.com/home/rtm.tcr.unittest/*
-// @include        https://*.www.rememberthemilk.com/home/rtm.tcr.unittest/*
+// @name           RTM: Tag Cloud Restructurer
+// @namespace      http://www.rememberthemilk.com/home/*
+// @include        http://www.rememberthemilk.com/home/*
+// @include        https://www.rememberthemilk.com/home/*
+// @include        http://*.www.rememberthemilk.com/home/*
+// @include        https://*.www.rememberthemilk.com/home/*
 // ==/UserScript==
 
 /*
@@ -134,24 +134,10 @@
 	                    color: 'blue' 
 	},
 
-	// 2-level hierarchy
-	{ prefix: '--',     type: sectionHierarchy, 
-	                    depth: 2,
-	                    sizes: ['6', '4'],
-	                    colors: ['green', 'purple', 'brown'] 
-	},
-
-	// 3-level hierarchy
 	{ prefix: '-',      type: sectionHierarchy, 
 	                    colors: ['green', 'purple', 'brown'] 
 	},
 
-	// 4-level hierarchy
-	{ prefix: '+',      type: sectionHierarchy, 
-	                    depth: 4,
-	                    sizes: ['6', '4', '3', '1'],
-	                    colors: ['green', 'purple', 'brown'] },
-
 	{ prefix: 'maybe',  type: sectionRename, 
 	                    displayname: 'Someday/Maybe', 
 	                    color: 'CornflowerBlue' 
