--- rtm-tcr.user.js	2012-04-05 21:41:54.000000000 -0400
+++ rtm-tcr-emalminator.user.js	2012-04-05 21:49:25.000000000 -0400
@@ -1,11 +1,11 @@
 // ==UserScript==
-// @name           RTM: Tag Cloud Restructurer
+// @name           RTM: Tag Cloud Restructurer (emalminator)
 // @version        1.1
-// @namespace      http://www.rememberthemilk.com/home/*
-// @include        http://www.rememberthemilk.com/home/*
-// @include        https://www.rememberthemilk.com/home/*
-// @include        http://*.www.rememberthemilk.com/home/*
-// @include        https://*.www.rememberthemilk.com/home/*
+// @namespace      http://www.rememberthemilk.com/home/emalminator/*
+// @include        http://www.rememberthemilk.com/home/emalminator/*
+// @include        https://www.rememberthemilk.com/home/emalminator/*
+// @include        http://*.www.rememberthemilk.com/home/emalminator/*
+// @include        https://*.www.rememberthemilk.com/home/emalminator/*
 // ==/UserScript==
 
 /*
@@ -51,9 +51,13 @@
 	borderColor: 'lightGrey',
 	hiddenTags: ['sent'],
 	renameTags: {
-		'-P': 'Personal',
-		'-W': 'Work',
-		'-P/Computer/TeX': 'TeX'
+		// *** emalminator START
+		'-a': 'Academic',
+		'-p': 'Personal',
+		'-pr': 'Professional',
+		'+goal': 'Goals',
+		'+project': 'Projects'
+		// *** emalminator END
 	},
 	matchTagCase: false
 };
@@ -68,7 +72,9 @@
 //   type's default preference dictionary below
 // - descriptions of preference keys are also given below with defaults
 
-var my_sections = [
+// *** emalminator START
+var emalminator_sections = [
+// *** emalminator END
 	{ prefix: 'inbox',  type: sectionSingle,
 	                    displayname: 'Unsorted',
 	                    color: 'orange'
@@ -79,9 +85,19 @@
 	                    color: 'red'
 	},
 
-	{ prefix: 'goal',   type: sectionSingle,
-	                    displayname: 'Goals',
-	                    color: 'black'
+	// *** emalminator START
+	{ prefix: 'system', type: sectionSingle,
+	                    displayOrder: -1,
+	                    displayname: 'System Notes',
+	                    color: '#444444'
+	},
+
+	{ prefix: '+',      type: sectionHierarchy,
+	                    depth: 2,
+	                    sizes: ['6', '4'],
+	                    separators: '+', 
+	                    colors: ['FireBrick', 'black', 'navy']
+	// *** emalminator END
 	},
 
 	{ prefix: '_',      type: sectionFlat,
@@ -89,12 +105,22 @@
 	                    color: '#444444'
 	},
 
+	// *** emalminator START
+	{ prefix: '¡',      type: sectionFlat,
+	                    displayname: 'Warnings',
+	                    color: 'fuchsia'
+	},
+
+	// *** emalminator END
 	{ prefix: '@',      type: sectionFlat,
 	                    displayname: 'Contexts',
 	                    color: 'blue'
 	},
 
 	{ prefix: '-',      type: sectionHierarchy,
+	                    // *** emalminator START
+  	                    separators: '/',
+	                    // *** emalminator END
 	                    colors: ['green', 'purple', 'brown']
 	},
 
@@ -106,13 +132,14 @@
 	// catch-all section for unprocessed lists and tags
 	{ prefix: '',       type: sectionFlat,
 	                    displayname: 'Miscellaneous',
-	                    runinText: false,
 	                    color: 'gray'
 	}
 ];
 
 // pick the above section list as sections to process
-var sections = my_sections;
+// *** emalminator START
+var sections = emalminator_sections;
+// *** emalminator END
 
 
 // default preferences for each type of section
