--- rtm_tcr_unittest.user.js	2012-04-04 17:31:08.000000000 -0400
+++ rtm_tcr.user.js	2012-04-04 17:28:58.000000000 -0400
@@ -1,11 +1,11 @@
 // ==UserScript==
-// @name           RTM: Tag Cloud Restructurer (rtm.tcr.unittest)
+// @name           RTM: Tag Cloud Restructurer
 // @version        1.1
-// @namespace      http://www.rememberthemilk.com/home/rtm.tcr.unittest/*
-// @include        http://www.rememberthemilk.com/home/rtm.tcr.unittest/*
-// @include        https://www.rememberthemilk.com/home/rtm.tcr.unittest/*
-// @include        http://*.www.rememberthemilk.com/home/rtm.tcr.unittest/*
-// @include        https://*.www.rememberthemilk.com/home/rtm.tcr.unittest/*
+// @namespace      http://www.rememberthemilk.com/home/*
+// @include        http://www.rememberthemilk.com/home/*
+// @include        https://www.rememberthemilk.com/home/*
+// @include        http://*.www.rememberthemilk.com/home/*
+// @include        https://*.www.rememberthemilk.com/home/*
 // ==/UserScript==
 
 /*
@@ -51,14 +51,9 @@
 	borderColor: 'lightGrey',
 	hiddenTags: ['sent'],
 	renameTags: {
-		// *** UNIT TEST START
-		'_rename-flat': 'Renamed flat tag',
-		'+h44/a1/b3': 'Renamed hierarchy tag',
-		'single-rename': 'Renamed single tag',
-		'NeXT': 'Next r&eacute;n&#x61;m&egrave;d',
-		'-h3A': 'Rename with exact case',
-		'-H3A': 'Rename with mixed case'
-		// *** UNIT TEST END
+		'-P': 'Personal',
+		'-W': 'Work',
+		'-P/Computer/TeX': 'TeX'
 	},
 	renameTagsMatchCase: false
 };
@@ -144,10 +139,7 @@
 	},
 
 	{ prefix: 'next',   type: sectionSingle, 
-	                    // *** UNIT TEST START
-	                    displayOrder: 1,
-	                    displayname: 'Next Actions (at top)', 
-	                    // *** UNIT TEST END
+	                    displayname: 'Next Actions', 
 	                    color: 'red'
 	},
 
@@ -156,58 +148,20 @@
 	                    color: 'black' 
 	},
 
-	// *** UNIT TEST START
-	{ prefix: 'single-rename',   type: sectionSingle, 
-	                    displayname: 'Should get overridden', 
-	                    color: 'black'
-	},
-	// *** UNIT TEST END
-
 	{ prefix: '_',      type: sectionFlat, 
 	                    displayname: 'Responsibilities', 
 	                    color: '#444444' 
 	},
 
-	// *** UNIT TEST START
-	{ prefix: '+_',     type: sectionFlat, 
-	                    displayname: 'Bottom List', 
-	                    color: '#444444',
-	                    displayOrder: -1
-	},
-	// *** UNIT TEST END
-
 	{ prefix: '@',      type: sectionFlat, 
-	                    // *** UNIT TEST START
-                        runinText: true,
-	                    // *** UNIT TEST END
 	                    displayname: 'Contexts', 
 	                    color: 'blue'
 	},
 
-	// *** UNIT TEST START
-	// 2-level hierarchy
-	{ prefix: '--',     type: sectionHierarchy, 
-	                    depth: 2,
-	                    sizes: ['6', '4'],
-	                    colors: ['green', 'purple', 'brown'] 
-	},
-	// *** UNIT TEST END
-
-	// *** UNIT TEST START
-	// 3-level hierarchy
-	// *** UNIT TEST END
 	{ prefix: '-',      type: sectionHierarchy, 
 	                    colors: ['green', 'purple', 'brown'] 
 	},
 
-	// *** UNIT TEST START
-	// 4-level hierarchy
-	{ prefix: '+',      type: sectionHierarchy, 
-	                    depth: 4,
-	                    sizes: ['6', '4', '3', '1'],
-	                    colors: ['green', 'purple', 'brown'] },
-
-	// *** UNIT TEST END
 	{ prefix: 'maybe',  type: sectionSingle, 
 	                    displayname: 'Someday/Maybe', 
 	                    color: 'CornflowerBlue' 
