--- rtm_tcr.user.js	2012-03-29 15:50:54.000000000 -0400
+++ rtm_tcr_emalminator.user.js	2012-03-29 15:53:39.000000000 -0400
@@ -1,10 +1,10 @@
 // ==UserScript==
-// @name           RTM: Tag Cloud Restructurer
-// @namespace      http://www.rememberthemilk.com/home/*
-// @include        http://www.rememberthemilk.com/home/*
-// @include        https://www.rememberthemilk.com/home/*
-// @include        http://*.www.rememberthemilk.com/home/*
-// @include        https://*.www.rememberthemilk.com/home/*
+// @name           RTM: Tag Cloud Restructurer (emalminator)
+// @namespace      http://www.rememberthemilk.com/home/emalminator/*
+// @include        http://www.rememberthemilk.com/home/emalminator/*
+// @include        https://www.rememberthemilk.com/home/emalminator/*
+// @include        http://*.www.rememberthemilk.com/home/emalminator/*
+// @include        https://*.www.rememberthemilk.com/home/emalminator/*
 // ==/UserScript==
 
 /*
@@ -47,6 +47,8 @@
 	borderColor: 'lightGrey',
 	hiddenTags: ['system', 'sent'],
 	renameTags: {
+		'+goal': 'Goals',
+		'+project': 'Projects'
 	}
 };
 
@@ -110,7 +112,7 @@
 // - ex: if '@' before '@_', '@' will grab all tags starting with '@'
 // overwrite per-section preferences here, too;
 
-var my_sections = [
+var emalminator_sections = [
 	{ prefix: 'inbox',  type: sectionRename, 
 	                    displayname: 'Unsorted', 
 	                    color: 'orange' 
@@ -121,9 +123,11 @@
 	                    color: 'red' 
 	},
 
-	{ prefix: 'goal',   type: sectionRename, 
-	                    displayname: 'Goals', 
-	                    color: 'black' 
+	{ prefix: '+',      type: sectionHierarchy, 
+	                    depth: 2,
+	                    sizes: ['6', '4'],
+	                    separators: '+', 
+	                    colors: ['FireBrick', 'black', 'navy'] 
 	},
 
 	{ prefix: '_',      type: sectionFlat, 
@@ -131,12 +135,18 @@
 	                    color: '#444444' 
 	},
 
+	{ prefix: '¡',      type: sectionFlat, 
+	                    displayname: 'Warnings', 
+	                    color: 'fuchsia' 
+	},
+
 	{ prefix: '@',      type: sectionFlat, 
 	                    displayname: 'Contexts', 
 	                    color: 'blue' 
 	},
 
 	{ prefix: '-',      type: sectionHierarchy, 
+  	                    separators: '/', 
 	                    colors: ['green', 'purple', 'brown'] 
 	},
 
@@ -153,7 +163,7 @@
 ];
 
 // pick the above section list as sections to process
-var sections = my_sections;
+var sections = emalminator_sections;
 
 /*
  * End of configuration section
