--- rtm_tcr.user.js	2012-03-27 22:22:40.000000000 -0400
+++ rtm_tcr_example.user.js	2012-03-27 22:40:17.000000000 -0400
@@ -1,10 +1,10 @@
 // ==UserScript==
-// @name           RTM: Tag Cloud Restructurer
-// @namespace      http://www.rememberthemilk.com/home/*
-// @include        http://www.rememberthemilk.com/home/*
-// @include        https://www.rememberthemilk.com/home/*
-// @include        http://*.www.rememberthemilk.com/home/*
-// @include        https://*.www.rememberthemilk.com/home/*
+// @name           RTM: Tag Cloud Restructurer (rtm.tcr.example)
+// @namespace      http://www.rememberthemilk.com/home/rtm.tcr.example/*
+// @include        http://www.rememberthemilk.com/home/rtm.tcr.example/*
+// @include        https://www.rememberthemilk.com/home/rtm.tcr.example/*
+// @include        http://*.www.rememberthemilk.com/home/rtm.tcr.example/*
+// @include        https://*.www.rememberthemilk.com/home/rtm.tcr.example/*
 // ==/UserScript==
 
 /*
