--- rtm_tcr.user.js	2012-03-30 16:34:15.000000000 -0400
+++ rtm_tcr_emalminator.user.js	2012-04-02 09:38:27.000000000 -0400
@@ -1,11 +1,11 @@
 // ==UserScript==
-// @name           RTM: Tag Cloud Restructurer
+// @name           RTM: Tag Cloud Restructurer (emalminator)
 // @version        1.1
-// @namespace      http://www.rememberthemilk.com/home/*
-// @include        http://www.rememberthemilk.com/home/*
-// @include        https://www.rememberthemilk.com/home/*
-// @include        http://*.www.rememberthemilk.com/home/*
-// @include        https://*.www.rememberthemilk.com/home/*
+// @namespace      http://www.rememberthemilk.com/home/emalminator/*
+// @include        http://www.rememberthemilk.com/home/emalminator/*
+// @include        https://www.rememberthemilk.com/home/emalminator/*
+// @include        http://*.www.rememberthemilk.com/home/emalminator/*
+// @include        https://*.www.rememberthemilk.com/home/emalminator/*
 // ==/UserScript==
 
 /*
@@ -46,8 +46,13 @@
 var globalprefs = {
 	drawSectionBorders: true,
 	borderColor: 'lightGrey',
-	hiddenTags: ['system', 'sent'],
+	hiddenTags: ['sent'],
 	renameTags: {
+		'-a': 'Academic',
+		'-p': 'Personal',
+		'-pr': 'Professional',
+		'+goal': 'Goals',
+		'+project': 'Projects'
 	},
 	renameTagsMatchCase: false
 };
@@ -117,7 +122,7 @@
 // - ex: if '@' before '@_', '@' will grab all tags starting with '@'
 // overwrite per-section preferences here, too;
 
-var my_sections = [
+var emalminator_sections = [
 	{ prefix: 'inbox',  type: sectionRename, 
 	                    displayname: 'Unsorted', 
 	                    color: 'orange' 
@@ -128,9 +133,17 @@
 	                    color: 'red'
 	},
 
-	{ prefix: 'goal',   type: sectionRename, 
-	                    displayname: 'Goals', 
-	                    color: 'black' 
+	{ prefix: 'system', type: sectionRename, 
+	                    displayOrder: -1,
+	                    displayname: 'System Notes', 
+	                    color: '#444444'
+	},
+
+	{ prefix: '+',      type: sectionHierarchy, 
+	                    depth: 2,
+	                    sizes: ['6', '4'],
+	                    separators: '+', 
+	                    colors: ['FireBrick', 'black', 'navy'] 
 	},
 
 	{ prefix: '_',      type: sectionFlat, 
@@ -138,12 +151,18 @@
 	                    color: '#444444' 
 	},
 
+	{ prefix: '¡',      type: sectionFlat, 
+	                    displayname: 'Warnings', 
+	                    color: 'fuchsia' 
+	},
+
 	{ prefix: '@',      type: sectionFlat, 
 	                    displayname: 'Contexts', 
 	                    color: 'blue'
 	},
 
 	{ prefix: '-',      type: sectionHierarchy, 
+  	                    separators: '/', 
 	                    colors: ['green', 'purple', 'brown'] 
 	},
 
@@ -160,7 +179,7 @@
 ];
 
 // pick the above section list as sections to process
-var sections = my_sections;
+var sections = emalminator_sections;
 
 /*
  * End of configuration section
